hazelcast/src/main/java/com/hazelcast/config/DurableExecutorConfig.java

```diff
@@ -57,6 +60,8 @@
 
     private String splitBrainProtectionName;
 
+    private boolean statisticsEnabled = true;
```

Comment:
I see we're missing this config in other places like XSD, YAML, XML and spring config (https://github.com/hazelcast/hazelcast/wiki/Checklist-for-Hazelcast-config-changes). Can you add it?

Reply:
done.